docs/DeveloperGuide.md

```diff
@@ -351,19 +375,19 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Deleting a recipe
 
-1. Deleting a person while all persons are being shown
+1. Deleting a recipe while all recipes are being shown
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+   1. Prerequisites: List all recipes using the `list` command. Multiple recipes in the list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `delete recipe -n pasta`<br>
```

Comment:
I think although we might enforce unique names for recipes, I believe we are still doing deletions by index?